records/board/hexiwear.yaml

```diff
@@ -0,0 +1,7 @@
+common:
+    sources:
+        board:
+            - source/board/hexiwear.c
+        target:
+            - source/target/freescale/rapid_iot/target.c
```

Comment:
Why is the rapid_iot target used for the hexiwear?

Reply:
Rapid-IoT is the next generation of Hewiwear.